integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -837,8 +837,6 @@ object CarbonDataRDDFactory {
                    s"${ carbonLoadModel.getDatabaseName }.${ carbonLoadModel.getTableName }")
```

Comment:
Just come across the style problems. Delete the write spaces after '{' and before '}'.

Reply:
fixed